src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -264,7 +267,8 @@ public Object addComponent(String componentName) {
   }
 
   public void checkMainTools() {
-    for (int i = 1; i < 17; i++) {
+    int wikitextShortcutsInsertSection = 17; //edit tools 'Insert' section in MediaWiki:Edittools
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)